tests/quic_p2p.rs

```diff
@@ -0,0 +1,40 @@
+use quic_p2p::{Config, Event, QuicP2p};
+use std::net::{IpAddr, Ipv4Addr};
+use std::sync::mpsc;
+use unwrap::unwrap;
+
+#[test]
+fn successfull_connection_stores_peer_in_bootstrap_cache() {
+    let (ev_tx, _ev_rx) = mpsc::channel();
```

Comment:
Maybe use local scope instead of variable shadowing (`ev_tx` is shadowed later):
```
let ... = {
  // some code
  QuicP2p::with_config(...)
};
```

It keeps surprises minimum if someone changed code.